src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+
+    @Test
+    void testFruitTransactionConstructorAndGetters() {
+        FruitTransaction.Operation operation = FruitTransaction.Operation.SUPPLY;
+        String fruit = "Apple";
+        int quantity = 10;
+
+        FruitTransaction transaction = new FruitTransaction(operation, fruit, quantity);
+
+        assertEquals(operation, transaction.getOperation());
+        assertEquals(fruit, transaction.getFruit());
+        assertEquals(quantity, transaction.getQuantity());
+    }
+
+    @Test
+    void testSetters() {
+        FruitTransaction transaction = new FruitTransaction(FruitTransaction.Operation.PURCHASE,
+                "Banana", 5);
+
```

Comment:
Don't test getters, setters, and constructors. It doesn't have any specific logic